gtsam/sfm/TranslationRecovery.h

```diff
@@ -28,6 +28,40 @@
 
 namespace gtsam {
 
+// Parameters for the Translation Recovery problem.
+class TranslationRecoveryParams {
```

Comment:
parameters are typically structs, with fields *without* `_` at end. This is now supported in wrapper, as well. That being said, I think betweenTranslations don't belong here.

Reply:
Yeah, I would prefer struct too, I used the class for support in wrapper. Good to know that structs are supported now :)